receiver/datadogreceiver/translator_test.go

```diff
@@ -89,25 +91,45 @@ func TestTracePayloadV05Unmarshalling(t *testing.T) {
 	assert.Equal(t, span.Name(), "my-resource")
 }
 
-func BenchmarkTranslator(b *testing.B) {
-
+func TestTracePayloadV07Unmarshalling(t *testing.T) {
+	var traces pb.Traces
 	payload, err := vmsgp.Marshal(&data)
-	assert.NoError(b, err)
-	dc := pb.NewMsgpReader(bytes.NewReader(payload))
-	defer pb.FreeMsgpReader(dc)
+	assert.NoError(t, err)
+	if err2 := traces.UnmarshalMsgDictionary(payload); err2 != nil {
+		t.Fatal(err2)
```

Comment:
```suggestion
	if err = traces.UnmarshalMsgDictionary(payload); err != nil {
		t.Fatal(err)
```
Minor, but you can reuse `err`.